galleries/examples/color/color_cycle_default.py

```diff
@@ -9,33 +9,27 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-prop_cycle = plt.rcParams['axes.prop_cycle']
-colors = prop_cycle.by_key()['color']
+from matplotlib.colors import TABLEAU_COLORS, same_color
+
 
-lwbase = plt.rcParams['lines.linewidth']
-thin = lwbase / 2
-thick = lwbase * 3
+def f(x, a):
+    return 0.85 * a * (1 / (1 + np.exp(-x)) + 0.2)
```

Comment:
I don't know what this is from reading it (though I do understand from the final image). Maybe could do with a comment or docstring.